android/src/main/kotlin/im/nfc/flutter_nfc_kit/FlutterNfcKitPlugin.kt

```diff
@@ -614,11 +560,8 @@ class FlutterNfcKitPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
             result.success(jsonResult.toString())
         }
 
-        try {
+        runOnNfcThread(result, "Enable reader mode") {
```

Comment:
You don't need. `pollTag` is called by `handleMethodCall` in `runOnNfcThread`, so just let the error propagate outside the function.

Reply:
You're right, removed.